tools/build-icons/utils/deprecationReasonTemplate.mjs

```diff
@@ -0,0 +1,17 @@
+export function deprecationReasonTemplate(
+  deprecationReason,
+  { componentName, iconName, toBeRemovedInVersion },
+) {
+  const removalNotice = toBeRemovedInVersion
+    ? ` This ${deprecationReason.startsWith('icon') ? 'icon' : 'alias'} will be removed in ${toBeRemovedInVersion}`
+    : '';
+
+  switch (deprecationReason) {
+    case 'alias.typo':
+      return `Renamed because of typo, use {@link ${componentName}} instead.${removalNotice}`;
+    case 'alias.naming':
+      return `This name is no longer compliant with our guidelines, use {@link ${componentName}} instead.${removalNotice}`;
```

Comment:
```suggestion
      return `This icon has been renamed because it is no longer compliant with our guidelines, use {@link ${componentName}} instead.${removalNotice}`;
```

Reply:
What about?

```suggestion
      return `The name of this icon had to be changed because it didn't meet our guidelines anymore, use {@link ${componentName}} instead.${removalNotice}`;
```

Reply:
Maybe instead of `had to be changed` do `was changed`.